src/sdk/clients/decorators/erc7579/erc7579.decorators.test.ts

```diff
@@ -85,28 +94,42 @@ describe("erc7579.decorators", async () => {
     expect(isK1ValidatorInstalled).toBe(true)
   })
 
-  test.skip("should uninstall a module", async () => {
-    const gas = await testClient.estimateFeesPerGas()
-
-    const hash = await nexusClient.uninstallModule({
-      ...gas,
+  test("should install a module", async () => {
+    const hash = await nexusClient.installModule({
       module: {
         type: "validator",
-        address: contracts.k1Validator.address,
-        context: "0x"
+        address: mockAddresses.MockValidator,
+        data: encodePacked(["address"], [eoaAccount.address])
       }
     })
 
     const { success } = await nexusClient.waitForUserOperationReceipt({ hash })
     expect(success).toBe(true)
   })
 
-  test.skip("should install a module", async () => {
-    const hash = await nexusClient.installModule({
+  test("should uninstall a module", async () => {
+    const [installedValidators, nextCursor] =
+      await nexusClient.getInstalledValidators({})
```

Comment:
Can you make this work without the empty bracket? I briefly tried and couldn't figure it out.

Reply:
Fixed